app/javascript/components/subject/MonitoringStatus.js

```diff
@@ -138,6 +141,15 @@ class MonitoringStatus extends React.Component {
     }
   }
 
+  handleKeyPress() {
```

Comment:
Is this in addition to clicking away (13 is enter, correct?)?

Reply:
13 is enter. The reason why this is here is because if you just press enter while focused on the text input, it will try to do a form submit (default behavior) and refresh the page (it doesn't actually submit) instead of having the modal show up (as if the user had clicked on "change jurisdiction")